### ISSUE TITLE
Bright exposure mode

### ISSUE THREAD

**Issue by Gerrie12345** [bitfocus/companion-module-sony-visca#51]
I noticed that Bright is not present in exposure mode options.
In the SRG-300SE OSD menu Bright is one of the exposure mode options.

**Comment by goseid** [comment]
You are correct. We haven't fully implemented the spec yet. Things have been added as needed or requested.  I'll add this soon.

**Comment by goseid** [comment]
@Gerrie12345 the SRG-300SE is included in the latest build and the Brightness feature is implemented for that camera. You'll need to use the latest beta to get that implementation. Companion v3.0.0+5899-beta or newer will have the current sony-visca release.

**Comment by Gerrie12345** [comment]
I installed v3.0.0+5906, but there is no Bright mode in the Exposure setting.

 

Van: Greg Oseid ***@***.*** 
Verzonden: donderdag 25 mei 2023 1:46
Aan: bitfocus/companion-module-sony-visca
CC: Gerrie12345; Mention
Onderwerp: Re: [bitfocus/companion-module-sony-visca] Bright exposure mode (Issue #51)

 

@Gerrie12345 <https://github.com/Gerrie12345>  the SRG-300SE is included in the latest build and the Brightness feature is implemented for that camera. You'll need to use the latest beta to get that implementation. Companion v3.0.0+5899-beta or newer will have the current sony-visca release.

**Comment by goseid** [comment]
Try choosing your camera model and then disabling the sony-visca extension and then re-enabling it. It seems to not update the settings until a reboot or re-enabling of the settings.

That is something I need to make better.

**Comment by Gerrie12345** [comment]
I disabled and enabled the connections, but nothing changed.

When I read the Help for Sony: VISCA v2.7.0 there is the Exposure Mode with his options: (auto/manual/shutter/iris/gain priority)

There should be also the bright option.

 

I tried the Set Brightness option, but the Brightness setting shows: ?? (11) (no options)

 

Van: Greg Oseid ***@***.*** 
Verzonden: zondag 28 mei 2023 15:03
Aan: bitfocus/companion-module-sony-visca
CC: Gerrie12345; Mention
Onderwerp: Re: [bitfocus/companion-module-sony-visca] Bright exposure mode (Issue #51)

 

Try choosing your camera model and then disabling the sony-visca extension and then re-enabling it. It seems to not update the settings until a reboot or re-enabling of the settings.

That is something I need to make better.

**Comment by Gerrie12345** [comment]
I have set my camera model en disabled and enabled the connection, there are now different values for Set Brightness, but it doesn't work until Bright mode is selected in Exposure mode. However, I still don't see a Bright option under Exposure.
